plugins/gitops-profiles/src/api.ts

```diff
@@ -79,6 +79,14 @@ export interface ListClusterRequest {
   gitHubToken: string;
 }
 
+export interface GithubUserInfoRequest {
+  accessToken: string;
```

Comment:
Let's get this in to fix the plugin, but imo it would be a bit cleaner to have the `GitOpsRestApi` depend on the `GithubAuth` api directly, i.e. passing it in via the constructor.